charts/trino/templates/configmap-worker.yaml

```diff
@@ -87,6 +96,24 @@ data:
   {{ $fileName }}: |
     {{- $fileContent | nindent 4 }}
 {{- end }}
+
+---
+
+{{- if .Values.accessControl }}{{- if eq .Values.accessControl.type "configmap" }}
```

Comment:
Can we reuse the configmap uses by the coordinator? If yes, extract it to a separate file. You can remove the `app.kubernetes.io/component` if it's shared.